### DISCUSSION TITLE
Zooz ZAC36 Titan Valve displaying unknown manufacturer/product code

### DISCUSSION THREAD

**Discussion by murkumurdok** [zwave-js/zwave-js#3414] [Q&A] [answered]
I am trying to add a ZAC36 water valve on zwavejs2mqtt 5.6.0/zwave-js 8.3.1, but the device keeps adding with unknown manufacturer data/product code.  I suspect this is because when the device's debug info is viewed, it displays the following:

`  "hexId": "0x027a-0x0101-0x0036",
  "dbLink": "https://devices.zwave-js.io/?jumpTo=0x027a:0x0101:0x0036:1.10"`

However, that is not the correct device id for the ZAC36, it should be 0x0270 instead of 0x027a.

I have tried excluding and including the device several times, both with security and without, and the device is successfully interviewed each time, it just comes back with the wrong info.  Any help would be appreciated.  It seems like the easiest fix would be to edit the device's hexid and dbLink, but I'm not sure where in zwavejs2mqtt I can edit that.

**Comment by robertsLando** [comment]
@blhoward2 @AlCalzone

**Comment by AlCalzone** [comment]
> it just comes back with the wrong info

That's the info reported **by the device** during the interview. What makes you believe that the manufacturer ID should be `0x0270`?

@marcus-j-davies any idea why the link https://devices.zwave-js.io/?jumpTo=0x027a:0x0101:0x0036:1.10 claims it is outside the version range of [this file](https://devices.zwave-js.io/?jumpTo=0x027a:0x0101:0x0036:0.0) which has a firmware range of 0.0-255.255?

**Reply by marcus-j-davies** [reply]
Mmm.
Let me investigate.

**Reply by marcus-j-davies** [reply]
@AlCalzone,
I should be showing a not found error, not a firmware range error - I can see where the problem is
Will see to it this evening - the firmware range error, was put in for the assistive search handling, and should not be seen for URL's

**Reply by AlCalzone** [reply]
Oh, right - the manufacturer ID is different for the one that exists. So a "not found" error would be correct.

**Reply by murkumurdok** [reply]
`0x270` is the id displayed in the zwavejs config db, so I assume that this is actually why the device is not identified correctly, because it doesn’t match up with an existing device in the db.

I figured it was an error with the device itself, but I’m not sure how to correct it short of a firmware update, which is curry available.

**Reply by AlCalzone** [reply]
The proper fix would be to submit a new device file with the correct manufacturer ID, so it gets recognized correctly.
/cc @blhoward2

**Reply by blhoward2** [reply]
Mine is a testing model so they may have changed the number in production. I'll confirm with Zooz.

**Reply by murkumurdok** [reply]
Thanks everyone

**Reply by marcus-j-davies** [reply]
Browser glitch fixed!

**Comment by shred86** [comment]
Just got one of these and noticed this issue as well. Glad to see it looks like it’s already been resolved.

As a side note, anyone else with this device getting correct temperature readings? Mine seems to be stuck at 88 F.

**Comment by shbatm** [comment]
What's the path forward on this? Does the file for ZAC36 created under https://github.com/zwave-js/node-zwave-js/pull/3269 need to have the manufactuer's id changed to `0x027a` in a new PR? It looks like the `0x0270` file still exists for the original Custos valve (`packages/config/config/devices/0x0270/bvs-zwu.json`) so changing [this](https://github.com/zwave-js/node-zwave-js/blob/8016cd7fb8732bc6005a9a1ce136ebc335a7ee27/packages/config/config/devices/0x027a/zac36.json#L3) should allow both to show up correctly.

**Comment by blhoward2** [comment]
It just needs to be copied into the 0x027a folder and then the manufactuerID needs to be changed there. There are some evidently with the 0x0270 manufacturer ID because I have one…

This was opened as a discussion and not an issue so I missed it when I was running through the issues…

**Reply by shbatm** [reply]
Looks like it's already copied (#3269), just never had the manufacturer id corrected. Am I seeing that correctly?

**Reply by blhoward2** [reply]
Yeah, I must have missed that while editing the file.

**Comment by blhoward2** [comment, marked as answer]
PR pushed: https://github.com/zwave-js/node-zwave-js/pull/3566
